dot/node_integration_test.go

```diff
@@ -154,78 +160,19 @@ func TestStartNode(t *testing.T) {
 	node, err := NewNode(cfg, ks)
 	require.NoError(t, err)
 
+	go func() {
+		<-node.started
+		node.Stop()
+	}()
 	err = node.Start()
 	require.NoError(t, err)
-	<-node.started
-	node.Stop()
```

Comment:
It seems that node.Start() blocks until node.Stop is called.  There is a WaitGroup in start that waits until Done is called in Stop, so I needed to put the <-node.started in a routine before calling node.Start because this test was getting stuck never moving past start.

Reply:
Ooof yeah weird piece of code there. It's a bit out of scope, but maybe rename it to `Run()` instead of `Start()`? Since it doesn't really start and return, it 'runs' all the services in a blocking way.